### DISCUSSION TITLE
Which speech recognition service do you prefer?

### DISCUSSION THREAD

**Discussion by kappaflow** [speech-translator-ext/speech-translator-readme#1] [Polls]
**[Google Speech-to-Text](https://www.google.com/intl/en/chrome/demos/speech.html) vs [Microsoft Speech-to-Text](https://speech.microsoft.com/portal/speechtotexttool)**

**Comment by blaues** [comment]
I wish there was a way to install your fantastic extension in my tablet. Is this possible?

**Reply by kappaflow** [reply]
Thank you for your feedback!

Yes, its possible to use it on Android. To do it you need to install a web browser that supports extensions, for example Kiwi Browser. Then install the extension from the store and start listening in a popup window.

Unfortunately, I don't think it's possible to use it with the Setup 1 configuration, because of the platform restrictions on audio devices access and lack of option of choosing audio devices in chromium. But you can still use it with your mic.

You might have a mic activation sound while using it on Android, but you can fix by enabling **Multiple audio devices** in **Sound assistant settings**. So you will be able to change the volume for that notification individually and basically mute it.

I will add an instruction for Android in the future versions of the extension.

**Reply by blaues** [reply]
Thank you so much! Your extension is truly one of a kind. I just wish there was a way to spread the word and promote your app because it's a lifesaver for me and there isn't anything even comparable. I watch a bunch of live programming from around the world and most of these programs don't contain captions.  The real time captions translated is excellent! Thank you so much for doing what you do!

**Reply by kappaflow** [reply]
I appreciate your feedback!
By the way, a big update will be released in the next few days. I also added some information on how the extension users can contribute to the project:
https://github.com/speech-translator-ext/speech-translator-readme/tree/main#contributing

**Comment by keyusa0000** [comment]
Hello support team. Thank you for this wonderful addition. This helped me a lot. Thank you. However, there is a small and simple problem in translation into Arabic. When you switch to the center text option, it does not change. Thank you.

**Reply by kappaflow** [reply]
Thank you for your feedback! I can confirm the issue that right-to-left aligned languages don't center correctly and I will fix it in the future releases.

**Reply by keyusa0000** [reply]
Thank you for the answer 👌

**Reply by kappaflow** [reply]
It has been fixed
